source/paths.d.ts

```diff
@@ -18,6 +18,18 @@ export type PathsOptions = {
 	@default 10
 	*/
 	maxRecursionDepth?: number;
+
+	/**
+	Style of the path.
+
+  @default 'a.0.b'
+	*/
+	style?: 'a[0].b' | 'a.0.b';
```

Comment:
```suggestion
	/**
	Use bracket notation for array indices.

	@default false

	@example
	```
	type Example = {
		array: string[];
	};

	type A = Paths<Example, {bracketNotation: false}>;
	//=> 'array' | 'array.0'

	type B = Paths<Example, {bracketNotation: true}>;
	//=> 'array' | 'array[0]'
	```
	*/
	bracketNotation?: boolean;
```

Reply:
Should we need to leave some buffer to expand this option for the future? like "a[0]['b']" style?

Reply:
No